### PR TITLE
helprank: reverse chronological order

### PR DESCRIPTION
**Pull request by raymondng**
Using counts to sort the results runs the risk of the results looking "stale" (if say one keep has many clicks & rekeeps it'll stay on top for a long time)

Based on input from @eishay , we will now use "reverse chronological" order (approximately, based on keep_id) so results will appear more "fresh" 

The trade-off is that your most "helpful" keep is no longer at the top. Ultimately we may want to let user decide how to view the results.

@yingjieMiao @stkem @eishay @JRuff42

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/model/KeepDiscoveryRepo.scala [hunk 1]
```diff
@@ -79,6 +82,10 @@ class KeepDiscoveryRepoImpl @Inject() (
     q.toMap()
   }
 
+  def getUriDiscoveriesWithCountsByKeeper(userId: Id[User], since: DateTime)(implicit r: RSession): Seq[(Id[NormalizedURI], Id[Keep], Id[User], Int)] = {
+    sql"select uri_id, keep_id, keeper_id, count(*) c from keep_click group by uri_id, keeper_id having keeper_id=$userId order by keep_id desc".as[(Id[NormalizedURI], Id[Keep], Id[User], Int)].list()
+  }
```
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepDiscoveryRepo.scala:87]: `since` not used?

Comment (raymondng) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepDiscoveryRepo.scala:87]: Good catch. Will fix.

**Comment by eishay** [comment]
great, thanks!

**Comment by yingjieMiao** [comment]
lgtm

**Pull request merged**